ext/standard/random.c

```diff
@@ -33,8 +33,11 @@
 #ifdef __linux__
 # include <sys/syscall.h>
 #endif
-#if defined(__OpenBSD__) || defined(__NetBSD__)
+#if defined(__OpenBSD__) || defined(__NetBSD__) || defined(__FreeBSD__)
 # include <sys/param.h>
```

Comment:
The FreeBSD version check should be present here, i guess. FreeBSD versions < 12 don't seem to include this header.

Thanks.

Reply:
In fact the version comes from this header.

Reply:
Indeed :)